arithmetization/src/main/java/net/consensys/linea/zktracer/opcode/OpCodeData.java

```diff
@@ -77,6 +77,14 @@ public boolean isHalt() {
     return this.instructionFamily == InstructionFamily.HALT;
   }
 
+  public boolean isCall() {
```

Comment:
We already have those method somewhere else (based on the opcode, not the opcode family)

Reply:
Yes. But it makes sense to have them defined on the instruction family level.

Reply:
I moved these methods to the `OpCodeData` level.